From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Ikey Doherty <michael.i.doherty@intel.com>
Date: Tue, 15 Mar 2016 16:46:42 +0000
Subject: [PATCH] Set default keyboard shortcuts for use within Clear Linux
 os-utils-gui

Signed-off-by: Ikey Doherty <michael.i.doherty@intel.com>

Upstream added a "<Primary><Alt>t" shortcut to run "exo-open --launch
TerminalEmulator", which should result in the same behavior as running
"xfce4-terminal".

Signed-off-by: Patrick McCarty <patrick.mccarty@intel.com>
---
 libxfce4kbd-private/xfce4-keyboard-shortcuts.xml | 1 +
 1 file changed, 1 insertion(+)

diff --git a/libxfce4kbd-private/xfce4-keyboard-shortcuts.xml b/libxfce4kbd-private/xfce4-keyboard-shortcuts.xml
index c22428e..c550443 100644
--- a/libxfce4kbd-private/xfce4-keyboard-shortcuts.xml
+++ b/libxfce4kbd-private/xfce4-keyboard-shortcuts.xml
@@ -29,6 +29,7 @@
       <property name="&lt;Super&gt;r" type="string" value="xfce4-appfinder -c">
         <property name="startup-notify" type="bool" value="true"/>
       </property>
+      <property name="Super_L" type="string" value="xfce4-popup-whiskermenu"/>
     </property>
   </property>
   <property name="xfwm4" type="empty">
